Sources/SpeziScheduler/Scheduler.swift

```diff
@@ -26,6 +26,12 @@ public class Scheduler<ComponentStandard: Standard, Context: Codable>: NSObject,
     private var initialTasks: [Task<Context>]
     private var cancellables: Set<AnyCancellable> = []
     
+    /// Indicates whether the necessary authorization to deliver local notifications is already granted.
+    public var localNotificationAuthorization: Bool {
+        get async {
+            await UNUserNotificationCenter.current().notificationSettings().authorizationStatus == .authorized
```

Comment:
One issue that we have here is that the Scheduler is not making a UI update in case these permissions are changed. We could call the `Observable`'s `objectWillChange` publisher when the user calls the `requestLocalNotificationAuthorization`. If the user changes the permissions in the settings app, we already have the Lifecycle methods that are used here that would update the UI and potentially refresh the state from here.

Reply:
Thanks for raising the issue, that's a good point! 🚀

I adjusted the implementation, please review it.